chain/epoch-manager/src/validator_selection.rs

```diff
@@ -202,15 +201,8 @@ pub fn proposals_to_epoch_info(
     // Select validators for the next epoch.
     // Returns unselected proposals, validator lists for all roles and stake
     // threshold to become a validator.
-    let validator_roles = if ProtocolFeature::StatelessValidation.enabled(protocol_version) {
-        select_validators_from_proposals(epoch_config, proposals, protocol_version)
-    } else {
-        old_validator_selection::select_validators_from_proposals(
-            epoch_config,
-            proposals,
-            protocol_version,
-        )
-    };
+    let validator_roles =
```

Comment:
I'm sorry it might not be straightforward to delete this as `proposals_to_epoch_info` is called for genesis epoch config as well. Let me take a look in a day or two if we can easily separate out the code as we did for genesis.

Reply:
[This PR](https://github.com/near/nearcore/pull/13256) should fix the problem!